apps/ledger-live-mobile/src/families/algorand/operationDetails.tsx

```diff
@@ -61,7 +62,8 @@ const OperationIcon = ({
   confirmed,
   operation: { hasFailed, extra },
 }: OperationIconProps) => {
-  const rewards = extra.rewards && extra.rewards.gt(0) ? extra.rewards : null;
+  const e = extra as AlgorandOperationExtra;
```

Comment:
You can change the type of operation in line 56 to `AlgorandOperation` and remove the as it should work fine

Reply:
Same problem basically

```
src/components/OperationIcon.tsx:54:9 - error TS2322: Type 'Operation' is not assignable to type 'AlgorandOperation'.
  Types of property 'extra' are incompatible.
    Type 'unknown' is not assignable to type 'AlgorandOperationExtra'.

54         operation={operation}
```